packages/ariakit/src/command/__examples__/command/style.css

```
@@ -0,0 +1,21 @@
+.button {
+  @apply
+    flex
+    h-10
+    cursor-pointer
+    items-center
+    justify-center
+    gap-1
+    rounded-lg
+    border-none
+    bg-primary-2
+    px-4
+    text-base
+    text-primary-2
+    hover:bg-primary-2-hover
+    focus-visible:ariakit-outline
+    aria-disabled:opacity-50
+    dark:bg-primary-2-dark
+    dark:text-primary-2-dark
+    dark:hover:bg-primary-2-dark-hover;
+}
```

If the styles are the same, we can just import them from the button example:
```suggestion
@import url("../../../button/__examples__/button/style.css");
```

---

I wasn’t sure about referencing into other components or not. Sounds good. Will fix.